### DISCUSSION TITLE
红隼识别多次

### DISCUSSION THREAD

**Discussion by 77221412** [MaaAssistantArknights/MaaAssistantArknights#11314] [General]
![屏幕截图 2024-12-08 120206](https://github.com/user-attachments/assets/11bf1209-2499-4104-986b-ecbf0e5355bb)
控制中枢的红隼每次都需要识别很多次，经常就在前五位但是不选择

**Comment by ABA2396** [comment]
大概率是被识别成红了，但你又不发日志谁知道是什么原因

**Comment by 77221412** [comment]
哦哦，好的。不好意思，我第一次用GitHub不知道日志是啥，等我再查找看看怎么发。
Sorry，this is my first time to use GitHub.Because I don't know what is “log”，I will search and send problem after a few days.



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年12月9日(周一) 凌晨1:43
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [MaaAssistantArknights/MaaAssistantArknights] 红隼识别多次 (Discussion #11314)




 
大概率是被识别成红了，但你又不发日志谁知道是什么原因
